src/psyclone/tests/gocean1p0_transformations_test.py

```diff
@@ -168,6 +168,7 @@ def test_loop_fuse_different_iterates_over():
                              newsched.children[1])
 
 
+@pytest.mark.xfail(reason="Needs a new way to produce Unexpected Exception")
```

Comment:
Please create a new issue or (better) fix this xfail.

Reply:
Fixed. I didn't realize last time that I had to update to `.children` to `.loop_body.children`

Reply:
Great.